test/fixtures/user-agents.json

```diff
@@ -0,0 +1,317 @@
+[
+  {
+    "desc": "Baiduspider",
+    "ua": "Mozilla/5.0 (compatible; Baiduspider/2.0; +http://www.baidu.com/search/spider.html)",
+    "expected": "bot"
+  },
+  {
+    "desc": "HubSpot Crawler",
+    "ua": "Mozilla/5.0 (compatible; HubSpot Crawler; +https://www.hubspot.com)",
+    "expected": "bot"
+  },
+  {
+    "desc": "PingdomPageSpeed",
+    "ua": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36 PingdomPageSpeed/1.0 (pingbot/2.0; +http://www.pingdom.com/)",
+    "expected": "bot:monitoring"
+  },
+  {
+    "desc": "AHC bot",
+    "ua": "AHC/2.1",
+    "expected": "bot"
+  },
+  {
+    "desc": "DataDog Synthetics",
+    "ua": "mozilla/5.0 (x11; linux x86_64) applewebkit/537.36 (khtml, like gecko) chrome/123.0.6312.122 safari/537.36 datadogsynthetics",
+    "expected": "bot:monitoring"
+  },
+  {
+    "desc": "Googlebot",
+    "ua": "Mozilla/5.0 (iPhone; CPU iPhone OS 6_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/6.0 Mobile/10A5376e Safari/8536.25 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)",
+    "expected": "bot:search"
+  },
+  {
+    "desc": "Googlebot Deepcrawl",
+    "ua": "Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html) https://deepcrawl.com/bot",
+    "expected": "bot:seo"
+  },
+  {
+    "desc": "Android Browser on Galaxy Nexus",
+    "ua": "Mozilla/5.0 (Linux; U; Android 4.0.2; en-us; Galaxy Nexus Build/ICL53F) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30",
+    "expected": "mobile:android:webkit"
```

Comment:
Surprising. Is this a real UA string?

Reply:
I guess so. It appears on many user-agent list

btw, I handpicked the user agent list from https://github.com/faisalman/ua-parser-js/blob/master/test/specs/browser-all.json